app/styles/pages/_mascots.scss

```diff
@@ -75,7 +75,7 @@
     min-width: inherit;
 }
 
-.mascots-faq {
+.mascots {
```

Comment:
To be able to reuse the content/sidebar combination already in use on the FAQ page I've renamed the class to be just `.mascots`. While it works the resulting sidebar is slightly larger than what is on the existing site.